src/trackers/ULCX.py

```
@@ -62,6 +64,49 @@ async def get_additional_checks(self, meta):
+            warned_once = False
+            audio_tracks = [track for track in mediainfo_tracks if track["@type"] == "Audio"]
+            for track_one in audio_tracks:
+                # This is technically O(n^2) but the number of audio tracks is usually very small
+                for track_two in audio_tracks:
+                    if (track_one["Format_Commercial_IfAny"] == "DTS-HD Master Audio" and track_two["Format_Commercial_IfAny"] != "DTS-HD Master Audio" and track_two["Format"] == "DTS") and (
+                        (track_one["Duration"], track_one["FrameRate"], track_one["FrameCount"], track_one["Language"]) ==
+                        (track_two["Duration"], track_two["FrameRate"], track_two["FrameCount"], track_two["Language"])
+                    ):
+                        if not warned_once:
+                            warned_once = True
+                            console.print(f"[bold red]DTS audio track appears to be a lossy duplicate of DTS-HD MA track.[/bold red]")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)